git-sync/main.go

```diff
@@ -168,12 +184,26 @@ func syncRepo(repo, dest, branch, rev string, depth int) error {
 
 	if *flChmod != 0 {
 		// set file permissions
-		_, err = runCommand("chmod", "", []string{"-R", string(*flChmod), dest})
+		_, err = runCommand("chmod", "", []string{"-R", string(*flChmod), pullTarget})
 		if err != nil {
 			return err
 		}
 	}
 
+	if atomic && strings.Compare(pullTarget, path.Join(dest, subdir)) != 0 {
+		args := []string{"-snf", pullTarget, "data-link"}
```

Comment:
Is there a benefit to declaring the args right before using them? That var doesn't seem to be used anywhere else.